pvlib/irradiance.py

```diff
@@ -243,6 +254,12 @@ def poa_horizontal_ratio(surface_tilt, surface_azimuth,
         Solar zenith angle.
     solar_azimuth : numeric
         Solar azimuth angle.
+    poa_projection_minimum : None or numeric, default None
+        Minimum allowable value for the projection for the solar angle
+        into the plane of the array. `None` applies no limit.
+    solar_zenith_projection_minimum : numeric, default None
+        Minimum allowable value for the projection of the solar angle
+        on to a horizontal surface. `None` applies no limit.
```

Comment:
I'm struggling to see how the `xxx_minimum` kwargs are more clear than handling the behind-the-plane or nearly co-planar cases directly, in the calculation of the ratios in the diffuse sky models.

Reply:
That's fair. I think this is a borderline case of "don't repeat yourself". I have no idea if people use these functions on their own.